packages/block-library/src/navigation-menu-item/edit.js

```diff
@@ -45,9 +45,7 @@ import { Fragment, useState, useEffect } from '@wordpress/element';
  * @param {Function} setter Setter attribute function.
  */
 const updateLinkSetting = ( setter ) => ( setting, value ) => {
-	if ( setting === 'newTab' ) {
-		setter( { opensInNewTab: value } );
-	}
+	setter( { opensInNewTab: value } );
```

Comment:
why not pick up the setting ID to update from the callback function instead of setting it explicitly? 

```es6
setter( { [ setting ]: value } );
```

So if we add another setting in the future such as `nofollow` for instance, we won't need to concern about propagating these values

Reply:
That's a copy/paste mistake by me! Sorry

Reply:
Fixed